src/cpp/shared/campaign/campaign.cpp

```diff
@@ -161,12 +161,12 @@ std::map<int, std::map<int, int>> xCampaignDifficulties = {
   {3, {{0, 0}, {1, 1}, {2, 2}, {3, 2}}}
 };
 
-std::map<int, std::map<int, std::vector<int>>> mapsToComplete;
+std::map<int, std::map<int, std::set<int>>> mapsToComplete;
```

Comment:
@Kert  Can you explain the problem this commit fixes?

Reply:
You can't have same values in set, so identical values won't pile up as they would do in vector. There are no clear functions used for any of those containers in our code.

Reply:
Why could it have identical values before?

Reply:
Oh, I missed your last sentence.